### PR TITLE
Utilities for Selenium project

### PR DESCRIPTION
**Pull request by sudheer51**

### CODE REVIEW

File: pluto/src/test/java/org/iit/mmp/tests/CarMake.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.iit.mmp.tests;
+
+public class CarMake {
+
+	public static void main(String[] args) {
+		
+		String[] cars = new String[5];
```
Comment (snehapatel3070) [inline pluto/src/test/java/org/iit/mmp/tests/CarMake.java]: try to use Array list instead of string

**Review by snehapatel3070 — CHANGES_REQUESTED** [review]

File: pluto/src/test/java/org/iit/mmp/tests/CarMake.java (hunk 1 above)
Comment (sudheer51) [reply pluto/src/test/java/org/iit/mmp/tests/CarMake.java]: I have implemented the ArrayList. Please check and review ASAP.

Comment (snehapatel3070) [reply pluto/src/test/java/org/iit/mmp/tests/CarMake.java]: ArrayList is implemented and closing the review comments.

**Review by snehapatel3070 — APPROVED** [review]
Approved

**Pull request merged**